README.md

```diff
@@ -422,12 +422,47 @@ with at least one node that has persistent memory device(s). For development or
 testing, it is also possible to use a cluster that runs on QEMU virtual
 machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
 
+- **Make sure that alpha feature gates CSINodeInfo, CSIDriverRegistry are enabled**
+
+The method to configure alpha feature gates may vary, depending on the kubernetes deployment.
+
 - **Label the cluster nodes that provide persistent memory device(s)**
 
 ```sh
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Verify that the node labels have been configured correctly**
+
+```sh
+    $ kubectl get nodes --show-labels
+```
+
+Output of above command should indicate that every non-master node has labels:
+```
+pmem-csi.intel.com/node=<NODE-NAME>,storage=pmem
+```
```

Comment:
Here we need to follow up with some explanation.

"If `storage=pmem` is missing, label manually as described above. If `pmem-csi.intel.com/node` is missing, then double-check that the alpha feature gates are enabled and the CSI driver is running."